src/main/resources/application-db.yml

```
@@ -0,0 +1,18 @@
+  sql:
+    init:
+      mode: ${MAKAO_DEPLOY_DB_INIT_MODE:always}
+      schema-locations: classpath:db/schema.sql
+  jpa:
+    hibernate:
+      ddl-auto: validate
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)